software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
+
+- Define what we (the FOSS United Community) wish to do. Ask ourselves 
+whether what we wish to do will help in full-fulling the community's goals.
+- Define your requirements for the software solution clearly. Clearly 
+differentiate between needs and wants. Needs are features that are crucial to 
+performing the task at hand and wants are niceties, things we can do without.
+- Look for FOSS solutions that are capable of meeting the needs at a level 
+that's acceptable to the community. Focus should be on the needs.
+- Look at the cost in terms of pricing, effort and time that the community needs 
+to spend to be able to use solution and constantly look for ways we can lower 
+such barriers and consider supporting the project if that helps.
+- When a solution is available in the FOSS ecosystem use it.
+- When such a solution is not available in the ecosystem assuming that this is 
+the only thing that prevents us from doing something that will benefit the cause 
+of FOSS United then look for proprietary solutions.
+- Assess the costs of each solution in terms of privacy, freedom, pricing, effort 
+and time that the community needs to spend to be able to use software and then 
+make a choice as a community.
+- All choices needs to be made in a democratic fashion. Enough time and effort 
+should be spent to form a collective decision as a community rather than make 
+decisions based on the opinions of a few people that happened to be online 
+when the issue was being discussed.
+
```

Comment:
As of now FOSS United does not have any formal membership model. The community is quite loosely tied. Not sure what "democratic" means in that context.

Reply:
It simply means that the decision making power shouldn't rest within the hands of a few.